### ISSUE TITLE
Retention Policy

### ISSUE THREAD

**Issue by sm2x** [Awesome-Technologies/synapse-admin#64]
Please include "rooms message retention policy" settings.

**Comment by awesome-michael** [comment]
Is there an endpoint in the admin api for that? Could you point out where this is documented? Unfortunately I don't know very much about retention policies.

**Comment by sm2x** [comment]
Will this help?
https://github.com/matrix-org/synapse/blob/master/docs/message_retention_policies.md#room-configuration

**Comment by awesome-michael** [comment]
I don't think we can include such a thing in near future. The retention policy is set by room admins, not by server admins. Therefore there is no endpoint in the admin api for that and we can't send events into rooms we're not in. Also it's still marked as experimental.

**Comment by dklimpel** [comment]
#108 was merged.
Can you check if you can find the information (`m.room.retention `) in the tab `State events`?
